estuary/api/v1.py

```diff
@@ -240,14 +242,22 @@ def get_siblings(resource, uid):
     :raises NotFound: if the item is not found
     :raises ValidationError: if an invalid resource was requested
     """
+    story_type_mapper = {'container': 'ContainerStoryManager', 'module': 'ModuleStoryManager'}
+    story_type = request.args.get('story_type', 'container').lower()
+
+    story_manager_class = story_type_mapper.get(story_type)
+    if story_manager_class:
+        story_manager = getattr(estuary.utils.story, story_manager_class)()
```

Comment:
Optional: You could make the values of `story_type_mapper` be the class objects so you'd avoid having to use `getattr`.